python/nano/src/bigdl/nano/pytorch/inference/optimizer.py

```diff
@@ -805,6 +805,10 @@ def quantize(model: nn.Module,
                                                                      calib_dataloader,
                                                                      input_sample)
 
+            # Add sample_size property to calib_dataloader to make the for loop in 
+            # `ipex_quantization_model.py` load sample items less than this value
+            calib_dataloader.sample_size = sample_size
```

Comment:
should we make this an private member with some ugly name to avoid duplicated member like `_nano_calib_sample_size` for example?